documentation/src/docs/controller-service.md

```diff
@@ -582,92 +569,88 @@ guarantee at least once processing.
 the metadata of streams, including the entire history of segments.
-Client accessing a stream need to contact the controller to obtain
+Client accessing a stream need to contact the Controller to obtain
 information about segments.
 
-Clients query controller in order to know how to navigate streams. For
-this purpose controller exposes appropriate APIs to get active segments,
-successors, predecessors and segment information and Uris. These queries
+Clients query Controller in order to know how to navigate streams. For
+this purpose Controller exposes appropriate APIs to get active segments,
+successors, predecessors and segment information and _URIs_. These queries
 are served using metadata stored and accessed via stream store
 interface.
 
 Controller also provides workflows to modify state and behavior of the
-stream. These workflows include create, scale, truncation, update, seal,
-and delete. These workflows are invoked both via direct APIs and in some
-cases as applicable via background policy manager (auto-scale and retention).
-
-<img src="./img/RequestProcessing.png" width="600" height="300" />
-<p align="center">
-  <i>Request processing</i>
-</p>
+stream. These workflows include _create, scale, truncation, update, seal,_
+and _delete_. These workflows are invoked both via direct APIs and in some
+cases as applicable via background policy manager (auto scale and retention).
+
 ![request processing](img/RequestProcessing.png)
+
 Request Processing Flow
 
-#### Create Stream<a name="createStream"></a>
+### Create Stream
 
-Create stream is implemented as a task on Task Framework. Create stream
+Create stream is implemented as a task on **Task Framework**. Create stream
 workflow first creates initial stream metadata with stream state set to
-*CREATING*. Following this, it identifies segment containers that should
-own and create segments for this stream and calls create-segment
+*Creating*. Following this, it identifies segment containers that should
+own and create segments for this stream and calls create segment
 concurrently. Once all create segments complete, the create stream task
-completes thus moving the stream to *ACTIVE* state. All failures are
+completes thus moving the stream to *Active* state. All failures are
 retried few times with exponential backoffs. However, if it is unable to
-complete any step, the stream is left dangling in *CREATING* state.
+complete any step, the stream is left dangling in *Creating* state.
 
-#### Update Stream
+### Update Stream
 
-Update stream is implemented as a task on Serialized Request
-Handler/Concurrent Event Processor framework. Update stream is invoked
-by an explicit API call into controller. It first posts an Update
-Request event into request-stream. Following that it tries to create a
+Update stream is implemented as a task on serialized sequest
+handler or concurrent event processor framework. Update stream is invoked
```

Comment:
` Update stream is implemented as a task on serialized sequest 
 handler` ~or~over `concurrent event processor framework. Update stream is invoked`

Reply:
Accepted suggestion